### ISSUE TITLE
Send PR for Sync Gateway Docker official README

### ISSUE THREAD

**Issue by tleyden** [couchbase/docker#52]
- [ ] Review README for marketing purposes (@jamiltz)
- [ ] Run READE through Docker tool to check for valid markdown (@ceejatec)
- [ ] If https://github.com/docker-library/official-images/pull/2082 has already been merged, then we can change "NOTE: until the official docker image is approved, use the staging repository via:" to point to the official Docker image
- [ ] Split the README into sections and enhance generator to cat into single README (@tleyden)
- [ ] Push README PR

**Comment by tleyden** [comment]
The upstream PR got merged: https://github.com/docker-library/official-images/pull/2082

**Comment by ceejatec** [comment]
No, it didn't actually - that was my fault, I guess. I proposed a different
change (adding 4.1.1 CE) from the same git branch, and *that* got merged,
which fooled this github PR into thinking it was merged too.

I will re-propose the sync gateway addition after making some of the
suggested changes.

Ceej
aka Chris Hillery

**Comment by tleyden** [comment]
Ok thanks for the clarification
